docs/user/project_metadata.md

```diff
@@ -0,0 +1,88 @@
+# Projects metadata
+
+
+## Project-URL
+
+A package can contain optional [Project URL](https://packaging.python.org/en/latest/specifications/core-metadata/#project-url-multiple-use) 
+keys.
+
+PyPI renders these URLs on the package project page. They are also available in 
+the JSON data when requesting the package information.
+
+### Verified details
+
+![Verified details](assets/verified_details.png){ loading=lazy }
+
+
+[Trusted Publishing](trusted-publishers) allow PyPI to verify certain 
+information submitted by the package maintainer. Verified information appears in
+a separate section of the package detail page. A verified URL will have a green 
+checkmark attached (:fontawesome-solid-circle-check:{ .checked }).
+
+!!! warning
+
+    An URL being verified only attest that the _provenance_ of the build has 
+    been verified, i.e. the packages sources are coming from the URL specified.
+
+    PyPI does not check the package content itself.
+
+
+### Icons
+
+![Unverified details](assets/unverified_details.png){ loading=lazy }
+
+While the labels can be arbitrary, `warehouse` recognizes the one from the
```

Comment:
```suggestion
While the labels can be arbitrary, PyPI recognizes the ones from the
```

Since this is user-facing documentation, we shouldn't use the codename 'warehouse' here.